### PR TITLE
Search coins

### PR DESCRIPTION
**Pull request by scottmillion**
Added search suggestions when typing in a search. The coinslist API is freaking ENOURMOUS though. You mentioned you had a better one to use custom that you made?

### CODE REVIEW

File: src/components/ChartLegendPrice/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+`
+
+export const ChartLegendPrice = (props) => {
+  return (
+    <ChartLegendPriceWrap>
+      <LegendNormal>BTC</LegendNormal>
+      <LegendLarge>
+        {props.currencySymbol}
+        {formatChartNumber(
+          props.data.filter((item) => item.id === 'bitcoin')[0].current_price,
```
Comment (yoloOnTheBattlefield) [inline src/components/ChartLegendPrice/index.js:20]: .find instead of filter

File: src/components/BitcoinLineChart/index.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      const dataPoints = []
+      const dataLabels = []
+      prices.forEach((price) => {
+        let myDay = new Date(price[0]).getDate().toString()
+        if (myDay.length === 1) {
+          myDay = '0' + myDay
+        }
+        dataLabels.push(myDay)
+        dataPoints.push(price[1])
+      })
```
Comment (yoloOnTheBattlefield) [inline src/components/BitcoinLineChart/index.js:68]: same here

File: src/components/BitcoinBarChart/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      const dataPoints = []
+      const dataLabels = []
+      volumes.forEach((volume) => {
+        let myDay = new Date(volume[0]).getDate().toString()
+        if (myDay.length === 1) {
+          myDay = '0' + myDay
+        }
+        dataLabels.push(myDay)
+        dataPoints.push(volume[1])
+      })
```
Comment (yoloOnTheBattlefield) [inline src/components/BitcoinBarChart/index.js:53]: const dataPoints = volumes.map((volume) => volume[1])
const dataLabels = volumes.map((volume) => new Date(volume[0]).getDate().toString())

then you can set state

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/ChartLegendPrice/index.js (hunk 1 above)
Comment (scottmillion) [reply src/components/ChartLegendPrice/index.js:20]: I fixed this on the pull request 2 as it's not part of this feature.

File: src/components/BitcoinLineChart/index.js (hunk 2 above)
Comment (scottmillion) [reply src/components/BitcoinLineChart/index.js:68]: I fixed this on the pull request 2 as it's not part of this feature.

File: src/components/BitcoinBarChart/index.js (hunk 3 above)
Comment (scottmillion) [reply src/components/BitcoinBarChart/index.js:53]: I fixed this on the pull request 2 as it's not part of this feature.

**Pull request closed without merging**